### DISCUSSION TITLE
"Turbo Tools" package

### DISCUSSION THREAD

**Discussion by davestewart** [vercel/turborepo#2148] [Show and tell]
Hello,

Loving Turborepo so far!

I wanted to make it a bit easier to use in day-to-day development, so I wrote an [Enquirer](https://www.npmjs.com/package/enquirer)-based UI to do some common tasks:

![screenshot](https://user-images.githubusercontent.com/132681/193613367-c5477ec2-3985-43fa-b1ed-a7d85742a84b.png)

I'm ready to publish; but I thought it would be polite to ask if the team is fine with me using the package name `turbo-tools` ?

~~The repo itself is [here](https://github.com/davestewart/turbo-tools)~~

EDIT: it's now [Spaceman](https://github.com/davestewart/spaceman), actually not dependent on Turbo Repo; install from [NPM](https://www.npmjs.com/package/@davestewart/spaceman)

Cheers,

Dave

**Comment by jaredpalmer** [comment]
Appreciate you reaching out. We indeed plan to use the the package name `turbo-tools` for future projects.  If you'd like, we'd be happy to credit you with the name grab.

**Reply by davestewart** [reply]
Hey Jared, 

No problem!

Perhaps I'll grab [turbo-tasks](https://www.npmjs.com/package/turbo-tasks) (which I think is actually more accurate) if you're good with that.

Anything else?

**Reply by davestewart** [reply]
Foiled!

- https://www.npmjs.com/package/turbotasks

I guess I'll stick it behind a namespace.

Unless you want to relinquish that name?

**Comment by davestewart** [comment]
@jaredpalmer - OK, I published to my own namespace:

- https://www.npmjs.com/package/@davestewart/turbo-tasks

I also shared on Twitter, so if you don't mind giving it a plug / RT / whatever that would be cool too:

- https://twitter.com/dave_stewart/status/1577670907404902400

**Reply by davestewart** [reply]
I just realised something funny – all this code I've written has _nothing_ to do with Turborepo!

It works for any monorepo setup (npm, pnpm, yarn) and doesn't even need Turborepo installed 🤦

Wood for the trees....
